int32 power_cmd
int32 motor_cmd
int32 voltage

int32 power_state
int32 motor_state
int32 position_feedback
